src/index.js

```diff
@@ -1,40 +1,96 @@
 import './styles/style.css';
+import Store from './modules/Store.js';
+import Elements from './modules/Elements.js';
+import Task from './modules/Task.js';
 
-const todoLists = [
-  {
-    index: 1,
-    desctiption: 'Play Guitar',
-    completed: true,
-  },
-  {
-    index: 2,
-    desctiption: 'Read ES6 syntax',
-    completed: false,
-  },
-  {
-    index: 3,
-    desctiption: 'Get Ready for WebPack',
-    completed: true,
-  },
-];
-
-const displayLists = () => {
-  const lists = todoLists;
-  const taskContainer = document.querySelector('.tasks-container');
-  lists.forEach((t) => {
-    const task = document.createElement('div');
-    task.innerHTML = `
+class Ui {
+  static displayTasks() {
+    const tasks = Store.getTasks();
+    tasks.forEach((task) => Ui.addTask(task));
+  }
+
+  static addTask(task) {
+    const elem = new Elements();
+    const { taskContainer } = elem;
+    const Element = document.createElement('div');
+    Element.innerHTML = `
     <div class="task-cont">
       <input type="checkbox" name="task" id="task-check-box">
-      <input type="text" value="${t.desctiption}" class="task-value">
+      <input type="text" value="${task.description}" class="task-value" id="${task.index}">
     </div>
     <div class="opp-icon">
       <i class="fa fa-ellipsis-v dots-icon"></i>
     </div>
     `;
-    task.classList.add('task');
-    taskContainer.appendChild(task);
-  });
-};
+    Element.classList.add('task');
+    taskContainer.appendChild(Element);
+  }
+
+  static clearInput() {
+    const elem = new Elements();
+    elem.inputField.value = '';
+  }
+
+  static changeIconTrash(icon) {
+    icon.className = '';
+    const deleteIconClass = ['fa', 'fa-trash', 'trash-icon'];
+    icon.classList.add(...deleteIconClass);
+  }
+
+  static changeIconDots(icon) {
+    icon.className = '';
+    const deleteIconClass = ['fa', 'fa-ellipsis-v', 'dots-icon'];
+    icon.classList.add(...deleteIconClass);
+  }
+
+  static taskActiveState(task) {
+    task.style.backgroundColor = '#f5f3ad';
+    Ui.changeIconTrash(task.lastElementChild.lastElementChild);
+  }
+
+  static taskClearActiveState(tasks) {
+    for (let i = 0; i < tasks.length; i += 1) {
+      tasks[i].style.backgroundColor = 'white';
+      Ui.changeIconDots(tasks[i].lastElementChild.lastElementChild);
+    }
+  }
+
+  static removeTask(task) {
+    const elem = new Elements();
+    const { taskContainer } = elem;
+    taskContainer.removeChild(task);
+  }
+}
+// Display tasks.
+Ui.displayTasks();
+
+// add Element
+let elem = new Elements();
+elem.inputField.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter' && elem.inputField.value !== '') {
+    event.preventDefault();
+    const task = new Task(Store.LASTINDEX + 1, elem.inputField.value, true);
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/Paul-tes/To-Do-List-App/pull/4#discussion_r1186026218), please make sure that all tasks have their `completed` set to `false` by default, and only set to true when the box has been checked 📌 
- [ ] Also, as suggested [here](https://github.com/Paul-tes/To-Do-List-App/pull/4#discussion_r1186029349) please make sure that after editing a task and reloading the page, the task maintains the new description and not the old one ✨

_See the video below for clarity 👇🎥_


https://user-images.githubusercontent.com/106119105/236503451-10ef2254-436a-4e1f-9228-2f3dca64467c.mp4

Reply:
sorry for wasting your time.